app/worker.py

```diff
@@ -113,6 +114,34 @@ def transcribe(
         "url_path": url_path,
     }
 
+@celery.task(name="detect_language", bind=True)
+def detect_language(self, audio_file_path: str, encode: bool):
```

Comment:
think i'd probably spend a bit of refactoring time in here if we went live with the associated plugin. works for illustrative purposes but could benefit from some error handling and code reuse.